### PR TITLE
longest_common_subsequence.cpp

### PR DESCRIPTION
**Pull request by sayantan1413**
Implemented the code of Longest common subsequence in C++

[x] - Code is well indented.
[x] - Code is well commented.
[x] - Code is not plagarized.
[x] - Code is placed in the proper folder.
[x] - The readme.md of the algorithm folder is updated with the code link.

### CODE REVIEW

File: Code/C++/longest_common_subsequence.cpp [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+* A subsequence is defined as the sequence that is same relative order but not neccesarily in a contiguous manner.
+* Problem Statement : Given two strings s1 and s2 and we have to find the longest common subsequence from both the string
+* Approach used : In this problem a dynamic programming solution will be used as the recursive algorithm have a overlapping
+  subproblem which can be solved by using dynamic programming. We can use both tabulation and memoization but we will be using 
+  tabulation in bottom up manner.
+* First we will be considering a 2D array of size (n+1) * (m+1) where n and m are hthe size of first and second string respectively
+  and we are taking an extra column and row to get the value of the substring of length 1 and 1 we need to have to value of 01, 00, 10
+* Any position at i,j i.e dp[i][j] will denote the length of longest common subsequence of prefix string of s1[0...i-1] and prefix
+  string of s2[0...j-1]
+* An example to demostrate the table view of the solution is
```
Comment (tarun26091999) [inline Code/C++/longest_common_subsequence.cpp:13]: remove line from 13 to 16

**Review by tarun26091999 — CHANGES_REQUESTED** [review]
everything else is fine, just make the asked changes

File: Code/C++/longest_common_subsequence.cpp (hunk 1 above)
Comment (sayantan1413) [reply Code/C++/longest_common_subsequence.cpp:13]: So I will remove the line from 13 to 28 and add it to the bottom of the code

**Comment by sayantan1413** [comment]
I have made the changes that I have removed the line 13 to 28 and added it to the bottom of the code !

File: Code/C++/longest_common_subsequence.cpp (hunk 1 above)
Comment (tarun26091999) [reply Code/C++/longest_common_subsequence.cpp]: yes

**Review by plazzy99 — APPROVED** [review]
LGTM 👍🏻

**Review by tarun26091999 — APPROVED** [review]
Looks good

**Comment by plazzy99** [comment]
#21 
@sayantan1413 remember to link the issue from next time

**Pull request merged**